scripts/awscli.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -e
 
-AWSCLI_VERSION="${AWSCLI_VERSION:-2.9.15}"
+AWSCLI_VERSION="${AWSCLI_VERSION:-2.11.15}"
```

Comment:
@mloberg is there a good way to test this? we tried looking at the dev build target for poco-prototype but weren't convinced we had a valid test (also task is sweet)